### PR TITLE
LPS-187908 Replace Draft application state by Unpublish state

### PR DESCRIPTION
**Pull request by LuismiBarcos**
Related ticket [LPS-187908](https://issues.liferay.com/browse/LPS-187908)
____
## Context
At the moment of this PR creation, there is no Draft feature implemented by Objects. So the Headless Builder does not support draft either because is based on Objects.

In order to avoid confusion the Draft application state is deleted and replaced by Unpublish state. Once the draft feature is implemented, the draft state will be added to the list of states.

### CODE REVIEW

**Comment by LuismiBarcos** [comment]
ci:test:sf

**Comment by LuismiBarcos** [comment]
ci:test:relevant

File: modules/apps/headless/headless-builder/headless-builder-impl/src/main/resources/com/liferay/headless/builder/internal/batch/headless-builder.json [hunk 1]
```diff
@@ -563,18 +563,18 @@
 							"value": {
 								"objectStates": [
 									{
-										"key": "draft",
+										"key": "published",
 										"objectStateTransitions": [
 											{
-												"key": "published"
+												"key": "unpublish"
```
Comment (4lejandrito) [inline modules/apps/headless/headless-builder/headless-builder-impl/src/main/resources/com/liferay/headless/builder/internal/batch/headless-builder.json:569]: `unpublished`?

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

File: modules/apps/headless/headless-builder/headless-builder-impl/src/main/resources/com/liferay/headless/builder/internal/batch/headless-builder.json (hunk 1 above)
Comment (4lejandrito) [reply modules/apps/headless/headless-builder/headless-builder-impl/src/main/resources/com/liferay/headless/builder/internal/batch/headless-builder.json:569]: both should end in `ed`.

**Comment by LuismiBarcos** [comment]
ci:stop

**Comment by LuismiBarcos** [comment]
ci:test:sf

**Comment by LuismiBarcos** [comment]
ci:test:relevant

**Review by 4lejandrito — APPROVED** [review]

**Comment by LuismiBarcos** [comment]
ci:forward

**Pull request closed without merging**